.github/workflows/test.yml

```diff
@@ -7,14 +7,27 @@ on:
 
 jobs:
   test:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        node-version: [ 16 ]
+      fail-fast: false
     name: Test
     runs-on: ubuntu-latest
     steps:
+    - name: Install dependencies
+      if: runner.os == 'macOS'
+      run: brew install libiconv
```

Comment:
Despite having a matrix set up, the matrix is not being referenced for L16, so all the tests are only being run on Ubuntu. Which also means that L18 is always skipped as well.

Reply:
I don't follow you. We test against all three OSes and one Node version (existing superstring doesn't work in newer Nodes, but updated superstring will). I'm probably missing your point, though.

Reply:
Ahahahaha. You're right. I'd have spotted this if I wasn't on my phone. We claim to be testing on all three OSes, but it's Ubuntu each time. Fixing now.